requirements/deploy.txt

```diff
@@ -406,7 +406,8 @@ structlog==22.1.0
     # via
     #   -r requirements/pip.txt
     #   django-structlog
-structlog-sentry==1.4.0
+    #   structlog-sentry
+structlog-sentry==2.0.0
```

Comment:
Changes at https://github.com/kiwicom/structlog-sentry/releases/tag/v2.0.0

Requires some changes to be able to migrate.

Reply:
Done at https://github.com/readthedocs/readthedocs-ops/pull/1230